cli_tools_e2e_test/diskinspect/go.mod

```diff
@@ -13,3 +14,5 @@ require (
 replace github.com/GoogleCloudPlatform/compute-image-tools/cli_tools => ../../cli_tools
 
 replace github.com/GoogleCloudPlatform/compute-image-tools/daisy => ../../daisy
+
+replace github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common => ../common
```

Comment:
hmm I didn't notice "replace" was added in a previous PR. Does it work as expected? I remember on build server it will be broken when I and Zoran did the similar thing.

Reply:
I've been testing this by building and running `gce_image_import_export_tests.Dockerfile` on cloud build, and there hasn't been a problem.

Reply:
Cool, no problem once it works. Let me hear more from you about why it's needed to make things work in tomorrow's sync up meeting.